docs/public/content/vuln_db/manuals/kb_importer.md

```diff
@@ -0,0 +1,152 @@
+# KB(Knowledge Base) Importer
+
+KB-Importer is a java application that is used to import vulnerabilities in Steady Backend. 
+
+The vulnerability database contains known vulnerabilities for which a fix exists. In fact, the fix (commit fix in the vulnerable library project) of the vulnerability must be provided in order to add a vulnerability to the database.
+
+Each vulnerability in the database is characterized by the set of program constructs that were changed in order to fix it.
+
+!!! info "Program Construct"
+        A program construct (or simply construct) is a structural element of the source code characterized by a type (e.g., `package`, `class`, `constructor`, `method`), a language (e.g., Java, Python), and a unique identifier (e.g., the fully-qualified name).
+
+!!! info "Construct Change"
+        A construct change is characterized by a change operation (i.e., addition, deletion or modification) on a construct. In case of modifications the vulnerability database also contains the abstract syntax tree (AST) of the construct before and after the fix, i.e., the AST of the vulnerable and fixed construct.
+
+[kb-importer](https://github.com/eclipse/steady/tree/master/kb-importer) is used to import such construct changes into the Steady backend.
+
+!!! info "Project KB"
+        [Project KB](https://github.com/SAP/project-kb) supports the creation, management and aggregation of a distributed, collaborative knowledge base of vulnerabilities that affect open-source software. Project-KB can be used to feed data to Eclipse Steady.
+
+## Usage
+To add a new vulnerability fix in a given library repository, the following fields need to be provided as input:
+
+* `directory (-d) <arg>` : A directory containing commits information for a vulnerability. More details regarding the directory structure is shown below.
+* `overwrite (-o)` : Overwrite the analysis of a vulnerability if it already exists in the backend.
+* `upload (-u)` : Upload construct changes. Without the -u option, the resulting JSON will be saved to file for you to review (e.g., to check that the list of construct changes is not empty).
+* `verbose (-v)` : Verbose mode to get more detailed logs.
+
+Other useful/required configurations:
+
+* `vulas.shared.backend.serviceUrl` : URL of the deployed rest-backend service (Mandatory).
+* `vulas.shared.cia.serviceUrl` : URL of the deployed rest-lib-util service (This is used if you are uploading the affected artifacts as shown in the below example).
+
+Directory structure should be as follows:
```

Comment:
Because the following might look scary to the user who thinks that they have to provide this exact structure by hand-crafting it, you could say upfront that this is created automatically by kaybee and add a forward pointer to the following section.